aiobotocore/response.py

```diff
@@ -152,6 +155,32 @@ def _verify_content_length(self):
     def tell(self):
         return self._self_amount_read
 
+    async def aclose(self) -> None:
```

Comment:
I was confused by the discussion, so please bear with me: Why do we need to provide `aclose`?